man/LassoDevelopment.Rd

```diff
@@ -29,11 +27,71 @@ F leads to removal of rows containing NULLs.}
 
 \item{debug}{Provides the user extended output to the console, in order
 to monitor the calculations throughout. Use T or F.}
+
+\item{cores}{Number of cores you'd like to use. Defaults to 2.}
+
+\item{modelName}{Optional string. Can specify the model name. If used, you must load the same one in the deploy step.}
 }
 \description{
 This step allows you to create a Lasso model, based on
 your data.
 }
+\section{Methods}{
+ 
+The above describes params for initializing a new lassoDevelopment class with 
+\code{$new()}. Individual methods are documented below.
+}
+
+\section{$new()}{
```

Comment:
The appearance of the $new() method shows up a little bit different than the other methods.
It should be `\section{\code{$new()}}{`